modules/fileout_netcdf/FONcTransform.cc

```diff
@@ -420,6 +422,13 @@ void FONcTransform::transform_dap2(ostream &strm) {
         uint64_t byteCount = 0;
 
         if (is_streamable()) {
+            // Verify the request hasn't exceeded bes_timeout.
+            RequestServiceTimer::TheTimer()->throw_if_timeout_expired("Ready to start streaming", __FILE__, __LINE__);
```

Comment:
I modified all the error message so that they start with bes-timeout: <message> in an attempt to tell the observer that the real cause was that the bes-timeout expired.